src/Ztm.WebApi/Controllers/ErrorController.cs

```diff
@@ -9,35 +9,65 @@ namespace Ztm.WebApi.Controllers
     public class ErrorController : ControllerBase
     {
         [Route("/error-development")]
-        public ActionResult ErrorDevelopment([FromServices] IHostingEnvironment webHostEnvironment)
+        public ActionResult ErrorDevelopment()
         {
-            var feature = HttpContext.Features.Get<IExceptionHandlerPathFeature>();
-            var ex = feature?.Error;
-            var isHttpResponseException = ex is HttpResponseException;
+            var info = HttpContext.Features.Get<IExceptionHandlerPathFeature>();
+            var ex = info?.Error;
+
+            int status;
+            string title;
+
+            if (ex is ApiException)
+            {
+                var apiEx = (ApiException)ex;
+
+                status = apiEx.Status;
+                title = $"{apiEx.Title} : {ex.GetType().Name}";
+            }
+            else
+            {
+                status = (int)HttpStatusCode.InternalServerError;
+                title = ex.GetType().Name;
+            }
 
             var problemDetails = new ProblemDetails
             {
-                Status = isHttpResponseException ? ((HttpResponseException)ex).Status : (int)HttpStatusCode.InternalServerError,
-                Instance = feature?.Path,
-                Title = isHttpResponseException ? $"{((HttpResponseException)ex).Title} : {ex.GetType().Name}" : ex.GetType().Name,
+                Status = status,
+                Instance = info?.Path,
+                Title = title,
                 Detail = $"{ex.Message} : {ex.StackTrace}",
             };
 
             return StatusCode(problemDetails.Status.Value, problemDetails);
         }
 
         [Route("/error")]
-        public ActionResult Error([FromServices] IHostingEnvironment webHostEnvironment)
+        public ActionResult Error()
         {
-            var feature = HttpContext.Features.Get<IExceptionHandlerPathFeature>();
-            var ex = feature?.Error;
-            var isHttpResponseException = ex is HttpResponseException;
+            var info = HttpContext.Features.Get<IExceptionHandlerPathFeature>();
+            var ex = info?.Error;
+
+            int status;
+            string title;
```

Comment:
We can remove the above 2 variables by moving instantiation of `ProblemDetails` here. e.g.:

```csharp
var details = new ProblemDetails()
{
    Instance = info?.Path,
    Detail = null
};
```

Reply:
I think current way is easier to read.